apparmor.d/groups/browsers/firefox

```diff
@@ -231,6 +240,7 @@ profile firefox @{exec_path} flags=(attach_disconnected) {
   owner /dev/tty@{int} rw, # File Inherit
 
   # Silencer
+  deny capability sys_ptrace,
```

Comment:
Might interfere with `crashreporter`? Anyway, I think telemetry should be treated as malicious.

Reply:
Remove this. You personal opinion on a software feature (even if I do agree with you here) should not interfere in the profile.

Reply:
Removed. I actually agree on this, as long as there's pre-compile/post-compile option (looking forward in the future).